runtime/parser2/expression.go

```diff
@@ -997,12 +1012,13 @@ func parseMemberAccess(p *parser, left ast.Expression, optional bool) ast.Expres
 
 func exprLeftDenotationAllowsNewline(tokenType lexer.TokenType) bool {
 
-	// The postfix force unwrap and invocation expressions don't support
-	// newlines before them, as this clashes with a unary negations and
-	// nested expressions on a new line / separate statement
+	// The postfix force unwrap, invocation expressions,
```

Comment:
Is this also true for the existing parser?

Something I think we should probably have in the Cadence language reference document (if it isn't there already).

Reply:
Yep, this is the same behavior as in the existing parser: 

- https://github.com/onflow/cadence/blob/master/runtime/parser/Cadence.g4#L459
- https://github.com/onflow/cadence/blob/master/runtime/parser/Cadence.g4#L461
- https://github.com/onflow/cadence/blob/master/runtime/parser/Cadence.g4#L553